### DISCUSSION TITLE
Is applications menu Wayland ready?

### DISCUSSION THREAD

**Discussion by ChildishGiant** [elementary/applications-menu#531] [General]
If not, [lightpad](https://github.com/libredeb/lightpad) was forked from slingshot and boasts Wayland compatibility so we could look there to see what changes they've made to allow for that.

**Comment by danirabbit** [comment]
It looks like that is a fork of a very very old version of the launcher. Like pre-0.2, so it wouldn't really be relevant to this codebase.

AFAIK the only thing to get away from here is LibPlank

**Comment by mcladams** [comment]
Is applications menu going to have some innovations more than gnome skin with square icons... please :)
